src/NeuralNetwork/NeuralNetworkData.js

```diff
@@ -839,8 +839,7 @@ class NeuralNetworkData {
       let modelMetadata = await axios.get(filesOrPath.metadata, {
         responseType: "text",
       });
-      modelMetadata = JSON.stringify(modelMetadata.data);
-      modelMetadata = JSON.parse(modelMetadata);
+      modelMetadata = JSON.parse(modelMetadata.data);
```

Comment:
If we want it as JSON then I think you could just remove the `responseType: "text"` and let `axios` parse it.  Try this:
```
const metadataRes = await axios.get(filesOrPath.metadata);
this.meta = metadataRes.data;
```

Reply:
Thank you @lindapaiste! @ziyuan-linn take a look at this feedback and after you make any changes as you see fit I can merge!